### ISSUE TITLE
Add ability for use custom image for search image.

### ISSUE THREAD

**Issue by SalahAdDin** [takeflight/wagtail-metadata#11]
Hi guys,

Thanks for this package.

I want ask you, will be possible use own custom image model for `search_image` field?

With the new version i had to create a new migrations, but, i think that will be great that we can use our custom image model:
```bash
Migrations for 'standard':
  standard/migrations/0003_auto_20160918_1647.py:
    - Alter field search_image on standardindexpage
    - Alter field search_image on standardpage
Migrations for 'blog':
  blog/migrations/0004_auto_20160918_1647.py:
    - Alter field search_image on blogindexpage
    - Alter field search_image on blogpage
Migrations for 'home':
  home/migrations/0002_auto_20160918_1647.py:
    - Alter field search_image on homepage
Migrations for 'work':
  work/migrations/0003_auto_20160918_1647.py:
    - Alter field search_image on workindexpage
    - Alter field search_image on workpage
Migrations for 'marketing':
  marketing/migrations/0002_auto_20160918_1647.py:
    - Alter field search_image on marketinglandingpage
Migrations for 'core':
  core/migrations/0003_auto_20160918_1647.py:
    - Alter field search_image on contactform
Migrations for 'services':
  services/migrations/0002_auto_20160918_1647.py:
    - Alter field search_image on servicespage
Migrations for 'about':
  about/migrations/0003_auto_20160918_1647.py:
    - Alter field search_image on aboutpage
Migrations for 'person':
  person/migrations/0002_auto_20160918_1647.py:
    - Alter field search_image on personindexpage
    - Alter field search_image on personpage
``` 

Thanks!

**Comment by timheap** [comment]
This is one for you, @SableWalnut. You can use the `wagtail.wagtailimages.models.get_image_model` function to get the correct model to use in the foreign keys. You will also have to update the migrations to use the `WAGTAILIMAGES_IMAGE_MODEL` setting - which is not guaranteed to exist.

**Comment by SableWalnut** [comment]
Closed with #15.
